testing/scripts/playwright_test.py

```
@@ -2247,6 +2250,520 @@ async def test_stylelint_clean(page: Page) -> None:  # noqa: ARG001
+        assert_true(f"{tid}: no overflow top",   not tip["overflowTop"],
+                    f"tipTop<0 for {tid}")
+        assert_true(f"{tid}: no overflow right",  not tip["overflowRight"],
+                    f"tipRight>vw for {tid}")
+        assert_true(f"{tid}: no overflow left",   not tip["overflowLeft"],
+                    f"tipLeft<0 for {tid}")
+
+    total_tips = len(results)
+    # 8 bubbles when sessions are disabled (vlsm-session bubble is conditional);
+    # 9+ when sessions are enabled.
+    assert_true(
+        f"found all expected help bubbles (≥8)",
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)